third_party/blink/renderer/modules/cache_storage/cache.cc

```diff
@@ -1273,6 +1278,7 @@ ScriptPromise Cache::KeysImpl(ScriptState* script_state,
     return promise;
   }
 
+  mojo::internal::AutoRecordReplayAssertBufferAllocations asserts("RUN-2445-3010");
```

Comment:
Do you not want different names (endings) for these asserts?

Reply:
Theoretically yes, but here, we'll just rely on the call stack. The bucketing will use the given label string, so it should not cause any problems 😅